portal_pages/tests/test_user_forms.py

```diff
@@ -0,0 +1,137 @@
+import datetime
+
+from django.test import TestCase
+
+from wagtail.wagtailcore.models import Page, Site
+
+from portal_pages.models import (
+    MarketplaceIndexPage, MarketplaceEntryPage,
+    CaseStudyIndexPage, CaseStudyPage,
+    BlogIndexPage, BlogPage
+    )
+
+
+class MarketplaceEntryPageAddTests(TestCase):
```

Comment:
This should either have a more generic name, or the test case methods need to be split out into 3 different tests...right now we're testing case study and blog also in this test.